packages/backend-dev-utils/src/ipcClient.ts

```diff
@@ -35,6 +35,12 @@ type Response =
       error: Error;
     };
 
+type ResponseHandler = (response: Response) => void;
+
+const isResponse = (message: unknown): message is Response => {
+  return typeof (message as Response)?.id === 'number';
+};
```

Comment:
Checking for `.type === responseType` is check we'd want to do here. It's more strict and avoids collisions with other messages that might also define a numeric `.id`.